src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+
+/**
+ * Represents a Person's block in the address book.
+ */
+
+public class Block extends AddressComponent {
+
+    public Block(String component) {
+        super(component);
+    }
+
```

Comment:
This is a redundant design: subclass has nothing new w.r.t. the superclass. If you really think these components share same features, you can simply declare `block, postalcode, street, unit` as type of `AddressComponent`.